### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by cebor** [webpack-contrib/extract-text-webpack-plugin#421]
Since the last update of `loader-utils@0.2.17` i'm getting the following deprecation warning:

```bash
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

ExtractTextWebpackPlugin: `^2.0.0-rc.3`

**Comment by bassbump** [comment]
Also see https://github.com/webpack/loader-utils/issues/56

**Comment by cebor** [comment]
fixed: 16e7601134a88bf9f983e79fb26484a9824432d5
